client/src/Components/CommonComponent/Sidebar/index.jsx

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Icon } from 'antd';
+
+import './style.css';
+
+export default class Sidebar extends Component {
+  state = {
+    accounts: 'hidden',
+    reports: 'hidden',
+    msg: 'hidden',
+    settings: 'hidden',
+    arrow_down_accounts:'inline',
+    arrow_up_accounts:'hidden',
+    arrow_down_reports:'inline',
+    arrow_up_reports:'hidden',
+    arrow_down_msg:'inline',
+    arrow_up_msg:'hidden',
+    arrow_down_settings:'inline',
+    arrow_up_settings:'hidden',
+  }
+  handleClick = (value) => (e) => {
+    this.setState((prev) => {
+      let display = '';
+      let up='';
+      let down='';
+      if (prev[value] === 'hidden') {
+        display = 'block';
+        up='inline';
+        down='hidden';
+      } else {
+        display = 'hidden';
+        up='hidden';
+        down='inline';
+      }
+      if(value==='accounts')
+      return { [value]: display , arrow_down_accounts:down,arrow_up_accounts:up};
+      if(value==='reports')
+      return { [value]: display , arrow_down_reports:down,arrow_up_reports:up};
+      if(value==='msg')
+      return { [value]: display , arrow_down_msg:down,arrow_up_msg:up};
+      if(value==='settings')
+      return { [value]: display , arrow_down_settings:down,arrow_up_settings:up};
+    })
+  }
+  render() {
+      
+    const { accounts, reports, msg, settings,arrow_down_accounts,arrow_up_accounts ,arrow_down_reports,arrow_up_reports,arrow_down_msg,arrow_up_msg,arrow_down_settings,arrow_up_settings} = this.state;
+    return (
+      <div className="container">
+
+        <div className="sidebar">
+          <h1 className="sidebar__header">زاجل</h1>
+          <div className="sidebar__bar">
+            <div className='home'>
+              <Link to="/" className="sidebar__links ">
+                <span>
+                  الرئيسية
+            <Icon type="home" style={{marginLeft:'7px'}}/>
+                </span>
+
+              </Link>
+            </div>
+            <div className='home'>
+              <Link to="/" className="sidebar__links" >
+                <span>
+                  إدارة الطلبات
+            {' '}
+                  <Icon type="menu-unfold" style={{marginLeft:'5px'}}/>
+                </span>
+
+              </Link>
+            </div>
+
+
+            <div className="sidebar__dropdown lists" onClick={this.handleClick('accounts')}>
+              إدارة الحسابات
+          {' '}
+
+              <Icon type="tool" />
+              {' '}
+              <span className={`${arrow_down_accounts}`}>
+            <Icon type="down" style={{marginLeft:'5px',fontSize:'15px'}}/>
```

Comment:
the arrow icon must be in the left side as in Figma

Reply:
but I see an Arabic control panel and the arrows put in the right side.

Reply:
this is an example for a dashboard in right-to-left: https://keenthemes.com/metronic/preview/demo1/rtl/
focus on the style of the sidebar the arrow on the left of the sidebar